ui/frame.ui

```diff
@@ -41,12 +41,15 @@
     <rect>
      <x>0</x>
      <y>0</y>
-     <width>100</width>
-     <height>23</height>
+     <width>50</width>
+     <height>50</height>
     </rect>
    </property>
+   <property name="accessibleName">
+    <string>pauseBtn</string>
+   </property>
    <property name="text">
```

Comment:
Explain, I use that property to skin certain things. I.e. the pause button in this case. 

`QPushButton[accessibleName="pauseBtn"]:focus` 

for example